junit-test/shared/src/test/scala/org/scalajs/junit/utils/JUnitTest.scala

```diff
@@ -236,21 +240,31 @@ abstract class JUnitTest {
 object JUnitTest {
   sealed trait Output
   final case class Log(level: Char, msg: String) extends Output
-  final case class Event(status: Status, testName: String) extends Output
+  final case class Event(status: Status, testName: String, throwableToString: Option[String]) extends Output
   final case class Done(msg: String) extends Output
 
   object Output {
+    // We need something that does not occur in test names nor in Throwable.toString.
+    val separator: String = "::"
```

Comment:
Consider
```suggestion
    private final val Separator = "::"
```
(`final` and no `: String` means it's a constant expression)